README.adoc

```diff
@@ -174,6 +174,13 @@ String jdbcUrl1 = cfEnvJdbc.findJdbcServiceByName("mysqlA").getUrl();
 String jdbcUrl2 = cfEnvJdbc.findJdbcServiceByName("mysqlB").getUrl();
 ----
 
+If you are creating a user-provided-service in order to connect to a database by jdbc, you will want to provide a uri property along with username and password in the credentials block.
+
+[source,bash]
+----
+cf cups sqlserver-ups -p '{ "uri": "sqlserver://hostname:1433;database=test_db", "username": "someuser", "password": "whatmeworry?" }'
```

Comment:
What user really wants to know is per database type(as those are different impls) what fields can go into a payload(like you can use `jdbcUrl`) and how those are processed into final connect string. We're going to use `2.2.0.RELEASE` as that's what we just tested.

Payload can have `port` or `dbname` so user need to know when those work and what is expected `uri` or `jdbccUrl` then.

Reply:
I believe the intention of the jdbc module impl is to avoid documenting every rdbms provider individially. For instance, the readme says "The method findJdbcService will heuristically look at all services for known tags, labels...".

Reply:
I've added a little more to this section. WDYT?